root/usr/share/nethesis/NethServer/Module/Mail/Domain/Modify.php

```diff
@@ -64,6 +69,21 @@ public function validate(\Nethgui\Controller\ValidationReportInterface $report)
         }
     }
 
+    public function readDkimFile()
+    {
+        if ( ! isset($this->parameters['domain'])) {
+            return '';
+        }
+        $fileName =  '/etc/opendkim/keys/' . $this->parameters['domain'] . '.txt';
+        $value = $this->getPhpWrapper()->file_get_contents($fileName, FALSE, NULL, -1);
```

Comment:
If we fix the dir permissions I'm afraid we do not have the rights to read that file directly and we need to implement a sudo helper script :(